source/04-PR/05.ptx

```diff
@@ -14,314 +14,313 @@
+          </statement>
         </task>
         <task>
           <statement>
-            <p>Find the degree of <m>f(x)</m>.
-            </p> <ol marker= "A.">
-          <li> <m>1</m></li>
-          <li> <m>2</m></li>
-          <li> <m>3</m></li>
-          <li> <m>6</m></li>
-        </ol> 
-        </statement>
-        <answer><p>D</p></answer>
+            <p>
+              Use the Remainder Theorem to confirm that your guesses are actually zeros.
+            </p>
+          </statement>
+        </task>
+        <task>
+          <statement>
+            <p>
+              Now use these zeros along with polynomial division to rewrite the function as <m>f(x)=(x-a)(x-b)q(x)</m> where <m>a</m> and <m>b</m> are zeros and <m>q(x)</m> is the remaining quadratic function.
+            </p>
+          </statement>
+        </task>
+        <task>
+          <statement>
+            <p>
+              Solve the quadratic <m>q(x)</m> to find the remaining zeros.
+            </p>
+          </statement>
         </task>
-        <task> 
-          <statement> 
-            <p>How do all of the exponents on the factors relate to the degree of <m>f(x)</m>?</p>
+        <task>
+          <statement>
+            <p>
+              List all zeros of <m>f(x)</m>.
+            </p>
+          </statement>
+        </task>
+        <task>
+          <statement>
+            <p>
+              Rewrite <m>f(x)</m> as a product of linear factors.
+            </p>
           </statement>
-          <answer>
-            <p>The degree is the sum of the exponents.</p>
-          </answer>
         </task>
-            <task> 
-              <statement> 
-                <p>How do the multiplicities of the zeros relate to the degree of <m>f(x)</m>?</p>
-              </statement>
-            </task>
-          <answer>
-            <p>Each exponent is the multiplicity of the corresponding zero.</p>
-          </answer>
-          </activity>  
+        
+      </activity>
 
+      <remark>
+        Using the graph to find an initial zero can be helpful, but they may not always be easy to identify.
+      </remark>
 
-<activity xml:id="write-polynomial-given-zeros">
-  
+   <activity xml:id="quad-rational-root">
+    <introduction>
+      <p>
+        Consider the quadratic function <m>f(x)=(2x-5)(3x-8)=6x^2-31x+40</m>.
+      </p>
+    </introduction>
+    <task>
+      <statement>
+        <p>
+          What are the roots of this quadratic?
+        </p>
+      </statement>
+    </task>
+    <task>
+      <statement>
+        <p>
+          What do you notice about these roots in relation to the factors of <m>a=6</m> and <m>c=40</m> in <m>f(x)=6x^2-31x+40</m>?
+        </p>
+      </statement>
+    </task>
+   </activity>
+   
+   <remark>
+    In <xref ref="quad-rational-root"/> we found that the roots were both factors of the constant term divided by factors of the leading coefficient.  This can be extended to polynomials of larger degree.
```

Comment:
The shortest proof I know of this is: If $\frac{p}{q}$ is a root of $a_nx^n+\cdots+a_1x+a_0$, then substituting in and clearing denominators gives $a_np^n+a_{n-1}p^{n-1}q+\cdots+a_1pq^{n-1}+a_0q^n=0$.  $p$ divides every term except $a_0q^n$, and $p,q$ are coprime, so $p$ divides $a_0$.  Similarly, $Q$ divides every term except $a_np^n$, so $q$ divides $a_n$.

Not sure if that is worth including though.